### PR TITLE
Introduced os.log to reduce complexity on logging errors

### PR DESCRIPTION
**Pull request by lojals**
#### What does this PR do? Any background context you want to provide?
We had a Logger implementation but we were using it in two places. This PR adds os.logs, a better solution for this kind of problem.
#### Where should the reviewer start?
`WLEmptyState/Classes/Misc/Swizzler.swift`

### CODE REVIEW

File: WLEmptyState/Classes/Misc/Logger.swift [hunk 1]
```diff
@@ -1,118 +0,0 @@
-//
-//  Logger.swift
-//  WLEmptyState
-//
-//  Created by Alexander Camacho on 2/6/19.
```
Comment (alexandercg) [inline WLEmptyState/Classes/Misc/Logger.swift:5]: 😢

**Review by alexandercg — APPROVED** [review]
LGTM 😢

File: WLEmptyState/Classes/Misc/Logger.swift (hunk 1 above)
Comment (saulurias) [reply WLEmptyState/Classes/Misc/Logger.swift:5]: 💔

**Review by saulurias — APPROVED** [review]
Nice 👍

**Review by leonardo-garcia — APPROVED** [review]
:shipit:

File: WLEmptyState/Classes/Misc/Logger.swift (hunk 1 above)
Comment (lojals) [reply WLEmptyState/Classes/Misc/Logger.swift:5]: 😢

**Pull request merged**